src/Configuration/Exception/FileCannotBeParsedAsYamlException.php

```diff
@@ -8,6 +8,15 @@
 
 final class FileCannotBeParsedAsYamlException extends RuntimeException
 {
+    public static function fromFilenameAndException(string $filename, RuntimeException $exception): self
+    {
+        return new self(sprintf(
+            'File "%s" cannot be parsed as YAML: %s',
+            $filename,
+            $exception->getMessage()
+        ));
+    }
+
     public static function fromFilename(string $filename): self
```

Comment:
Is this method then still in use?

Reply:
jep, https://github.com/qossmic/deptrac/search?q=fromFilename

Reply:
it seems it is only used from within a test. not sure it make sense to delete it though for BC reasons.

Reply:
There is no BC consideration if there is no public user-facing API, based on the usage of this tool. So from that perspective, it should be fine to delete it. The fact that it is used only for testing is problematic and IMHO symptom of bad design. If you are able to rewrite the test to not use it, it is IMHO safe to delete.

Reply:
ok, removed the method and adjusted the test-case for it to actually test the newly introduced method